lldb/source/Plugins/Process/Utility/StopInfoMachException.cpp

```diff
@@ -491,14 +491,13 @@ static StopInfoSP GetStopInfoForHardwareBP(Thread &thread, Target *target,
                                            uint64_t exc_sub_sub_code) {
   // Try hardware watchpoint.
   if (target) {
+    // LWP_TODO: We need to find the WatchpointResource that matches
```

Comment:
`LWP_TODO` doesn't mean much to a lot of contributors, maybe you can change this to something like `TODO(Large Watchpoint Support)`? Mostly to avoid acronyms that aren't written down anywhere else.

Reply:
Yeah good point my idea was that these are temporary placeholders as I roll this out, so they wouldn't be in the source for very long. (this feature is not really useful with all the parts landed so it's unlikely I'll be distracted by some other issue ....)